test/controllers/chargeD/DeleteMemberControllerSpec.scala

```diff
@@ -42,24 +44,30 @@ import uk.gov.hmrc.viewmodels.{NunjucksSupport, Radios}
 import scala.concurrent.Future
 
 class DeleteMemberControllerSpec extends ControllerSpecBase with MockitoSugar with NunjucksSupport with JsonMatchers with OptionValues with TryValues {
+  private val mutableFakeDataRetrievalAction: MutableFakeDataRetrievalAction = new MutableFakeDataRetrievalAction()
   private val mockAftConnector: AFTConnector = mock[AFTConnector]
+  private val application: Application = applicationBuilderMutableRetrievalAction(mutableFakeDataRetrievalAction)
+    .overrides(
+      bind[AFTConnector].toInstance(mockAftConnector)
```

Comment:
You could just pass the extra binding into the extraModules parameter of the data retrieval action, i.e. applicationBuilderMutableRetrievalAction(mutableFakeDataRetrievalAction, Seq(bind[AFTConnector].toInstance(mockAftConnector)))

Reply:
Done